### DISCUSSION TITLE
RVX settings icon

### DISCUSSION THREAD

**Discussion by sr-shishir** [anddea/revanced-patches#299] [Ideas]
If possible then add ReVanced Extended icon changed automatically in Settings according to my selection of “Custom Branding Icon”. 🙂 or you can keep the similar “line icons” like the third image.

![IMG_20240410_103616](https://github.com/anddea/revanced-patches/assets/139035194/d79a13a3-23a6-4228-b296-91fd3dd9692b)

**Comment by anddea** [comment]
Implemented in last releases.

**Reply by sr-shishir** [reply]
Thanks
